infra/k8s/testground-infra/values.yaml

```diff
@@ -85,3 +92,26 @@ redis:
       namespace: default
   cluster:
     enabled: false
+  usePassword: false
+  securityContext:
+    - name: net.core.somaxconn
+      value: 100000
+    - name: net.netfilter.nf_conntrack_max
+      value: 100000
+  master:
+    extraFlags:
+    - "--maxclients 100000"
+    podAnnotations:
+       cni: flannel
+    resources:
+      requests:
+        memory: 13000Mi
+        cpu: 6500m
+  sysctlImage:
```

Comment:
I noticed that this was previously applied to `slave` pods, so it was essentially redundant change from my part.

Do you know if it is necessary now, specially when we have the `securityContext` at the top? We should look into the actual chart, and see if this is applied or not.

Reply:
According to the comments in the helm chart, a sufficiently high version of kubernetes makes this redundant, and I removed the sysctlImage in a later commit.